cmd/account/governance.go

```diff
@@ -0,0 +1,193 @@
+package account
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/spf13/cobra"
+
+	governance "github.com/oasisprotocol/oasis-core/go/governance/api"
+	upgrade "github.com/oasisprotocol/oasis-core/go/upgrade/api"
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/connection"
+
+	"github.com/oasisprotocol/cli/cmd/common"
+	cliConfig "github.com/oasisprotocol/cli/config"
+)
+
+var (
+	govCmd = &cobra.Command{
+		Use:     "governance",
+		Short:   "Governance operations",
+		Aliases: []string{"gov"},
+	}
+
+	govSubmitProposalCmd = &cobra.Command{
+		Use:   "submit-proposal",
```

Comment:
Could we simply call it *submit*?

```suggestion
		Use:   "submit",
```

Reply:
Now it is "create proposal", we could also have just "create" but then it is less clear what is being created (only though a proposal is currently the only thing that can be created).